src/System.Runtime/tests/TestStructs/System.TestStructs.ilproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.IL">
+  <PropertyGroup>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <ProjectGuid>{6B002B34-089C-4BC4-91DD-57D350DEA91C}</ProjectGuid>
+    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+    <TestStrongNameKeyId>Microsoft</TestStrongNameKeyId>
```

Comment:
Is the "TestStrongNameKeyId" actually needed?

Reply:
Yes, without it I'm getting `Error: ilasm on CoreCLR does not support full sign.`